sentry-backtrace/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+//! Backtrace Integration and utilities for sentry.
+//!
+//! Exposes functions to capture, process and convert/parse stacktraces, as well
+//! as integrations to process event stacktraces.
+
+#![deny(missing_docs)]
+
+mod integration;
+mod parse;
+mod process;
+mod trim;
+mod utils;
+
+pub use crate::integration::{
+    current_thread, AttachStacktraceIntegration, ProcessStacktraceIntegration,
+};
+pub use crate::parse::parse_stacktrace;
+pub use crate::process::{backtrace_to_stacktrace, process_event_stacktrace};
+pub use crate::trim::trim_stacktrace;
+pub use crate::utils::error_typename;
+pub use sentry_core::protocol::{Frame, Stacktrace};
```

Comment:
Out of curiosity, why are you re-exporting this? Users should always get this from `sentry::protocol`

Reply:
Just for convenience, so the crate exposes all the non-std types it consumes/returns. Same reason that `sentry-types` re-exports a bunch of crono/uuid types I guess.